lib/policies/terminate/terminate.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function (params) {
+  params.statusCode = Number(params.statusCode) || 400;
+  params.message = params.message || 'Terminated';
+  return function (req, res, next) {
+    return res.status(params.statusCode).send(params.message);
```

Comment:
I would put the default values in the schema instead of the code. `ajv` fully supports [default values](https://github.com/epoberezkin/ajv#assigning-defaults) and will help the UI as well when using it around, as all the required data is going to be there.

Reply:
but what if I don't use API for configuration ? 
It will not generate settings based on that